corehq/tabs/tabclasses.py

```diff
@@ -1437,7 +1437,7 @@ def sidebar_items(self):
         if user_is_admin:
             items.append((_('Project Administration'), _get_administration_section(self.domain)))
 
-        if self.couch_user.can_edit_motech:
+        if self.couch_user.has_permission(self.domain, 'edit_motech'):
```

Comment:
I noticed that `self.couch_user.can_foo` is truthy for any "foo". That doesn't make sense to me. I'll follow up.

Reply:
Good to know! I feel a bit ambivalent about how opaque this is, but the result is more readable, so I'll go with it.